HelpMyStreetFE/HelpMyStreetFE/src/sass/components/profile/_job.scss

```diff
@@ -470,6 +526,37 @@
             }
         }
 
+        &.offline-details-container {
+            @media screen and (max-width: $med-screen) {
+                display: none;
```

Comment:
Not sure about hiding this on mobile devices...  I guess printing won't work so well, but some users might still want to trigger an email...